firefox_ui_tests/remote/security/test_safe_browsing_warning_pages.py

```diff
@@ -83,12 +83,12 @@ def check_report_button(self, unsafe_page):
 
         # Wait for the button to become stale, then wait for page load
         # so we can verify the url even if a redirect happens
-        self.wait_for_condition(expected.element_stale(button))
+        Wait(self.marionette).until(expected.element_stale(button))
 
         # TODO: Bug 1140470: use replacement for mozmill's waitforPageLoad
-        self.wait_for_condition(lambda mn: mn.execute_script("""
-          return document.readyState == 'complete';
-        """))
+        Wait(self.marionette).until(lambda mn: (
+            mn.execute_script("""return document.readyState == 'complete';"""))
```

Comment:
If you have the content of `execute_script` in a single line, it's enough to use a single quote sign. Feel free to fix it, but that's optional.